libs/partners/ollama/langchain_ollama/llms.py

```diff
@@ -24,15 +24,93 @@
 
 
 class OllamaLLM(BaseLLM):
-    """OllamaLLM large language models.
-
-    Example:
+    """Ollama large language models.
+
+    Setup:
+        Install ``langchain-ollama`` and install/run the Ollama server locally:
+
+        .. code-block:: bash
+
+            pip install -U langchain-ollama
+            # Visit https://ollama.com/download to download and install Ollama
+            # Then start the server: ollama serve
+
+        Download a model to use:
+
+        .. code-block:: bash
+
+            ollama pull llama3.1
+
+    Key init args — generation params:
+        model: str
+            Name of the Ollama model to use (e.g. "llama3.1", "llama2").
```

Comment:
```suggestion
            Name of the Ollama model to use (e.g. ``'llama4'``).
```

Tend to prefer newer models. Also formatting.

Reply:
(unless there's a compat issue with `llama4` I'm not aware of)

Reply:
I'll update the model example to 'llama4' and adjust the formatting with backticks as you suggested.